index.html

```diff
@@ -42,16 +86,18 @@ <h1 class="heading">Capital Jazz Fest 2023</h1>
             </div>
         </section>
     </header>
-    <section class="section2">
+    <section class="section2" id="program">
         <div class="title-container">
             <h2 class="title">Main Program</h2>
```

Comment:
- [ ] This line under this text should be a red line and not a colorless rectangle as you have in your design. Please fix this. Please make the width as small as what you have in the design. 
![image](https://user-images.githubusercontent.com/84156714/212482111-d5941a6a-aedc-4e31-90c5-11b289f2d68f.png)

- [ ] This section has some transparent box behind the various items. I see that your logos are way too big. Kindly reduce their size.
![image](https://user-images.githubusercontent.com/84156714/212482355-9db0ec6c-5073-4469-9079-96a069d8cc0d.png)

### Ultimately, you need to implement this portion of the project again and make it look as much like the template.

Reply:
- [ Optional] As requested by the previous reviewer kindly add a transparent background to the elements of the main program section. You can achieve it by adding for example a `background_color: rgba(211, 211, 211, 0.2)` to every `li`.